apps/dashboard/src/app/components/topbar/topbar.component.ts

```diff
@@ -19,46 +17,16 @@ export class TopbarComponent implements AfterViewInit {
   userInfo = this.userService.userInfo;
   kb = this.sdk.currentKb;
   isStage = location.hostname === 'stashify.cloud';
-  searchEnabled = this.appService.searchEnabled;
-
-  searchWidget = this.kb.pipe(
-    distinctUntilKeyChanged('id'),
-    tap(() => {
-      document.getElementById('search-widget')?.remove();
-    }),
-    switchMap((kb) => kb.getLabels().pipe(map((labelSets) => ({ kb, labelSets })))),
-    map(({ kb, labelSets }) => {
-      const hasLabels = Object.keys(labelSets).length > 0;
-      const features = !hasLabels
-        ? DEFAULT_FEATURES_LIST.split(',')
-            .filter((feature) => feature !== 'filter')
-            .join(',')
-        : DEFAULT_FEATURES_LIST;
-      return this.sanitized.bypassSecurityTrustHtml(`<nuclia-search id="search-widget" knowledgebox="${kb.id}"
```

Comment:
We do want to remove the widget from the topbar, but at the moment that's how we get the viewer working in resources list.
So with this change it will be broken.
You need to inject the widget viewer (which is a custom element too) in resources list page.